src/utils/legacyFPD.js

```diff
@@ -0,0 +1,48 @@
+import {deepAccess, mergeDeep} from '../utils.js';
+
+/**
+ * Returns backwards compatible FPD data for modules
```

Comment:
my only thought here is if it turns out only one adapter is using, eg inmar, would it be better to put this in the inmar adapter itself?

Reply:
yes, that would avoid extra weight for anyone who's not using inmar. I'll update if it turns out that luponmedia does not need the legacy conversion logic. Ideally, inmar would also drop the old format eventually.